### PR TITLE
Project and blogs update

### PR DESCRIPTION
**Pull request by tarunyadav1**
Hey, 

Project and blogs update. Kindly accept my request.

thank you

### CODE REVIEW

File: content/participants/ttjc@tarunyadav1/index.md [hunk 1]
```diff
@@ -34,8 +34,18 @@ I need a mentor to create connections in the tech industry.
 
 ### projects
 
-Coming soon...
+#### **FIRST PROJECT**
+
+    A Movie Library App made with ReactJS using The Movie Database API.
```
Comment (khusharth) [inline content/participants/ttjc@tarunyadav1/index.md:39]: Hey! @tarunyadav1 Please remove space between the left margin and your text in your project and blog description. Using tabs in markdown converts the normal text into code blocks.

Comment (tarunyadav1) [reply content/participants/ttjc@tarunyadav1/index.md:39]: Thank you @khusharth. I will remove Tabs. Do I need to create another pull request?

Comment (khusharth) [reply content/participants/ttjc@tarunyadav1/index.md:39]: No. You can just add another commit in this branch.

Comment (tarunyadav1) [reply content/participants/ttjc@tarunyadav1/index.md]: @khusharth  I had remove Tab. Kindly accept my request. Once again thank you.

File: content/participants/ttjc@tarunyadav1/index.md [hunk 2]
```diff
@@ -36,16 +36,16 @@ I need a mentor to create connections in the tech industry.
...
-    Github link: https://github.com/tarunyadav1/Movie-Library-App-using-Reactjs-and-The-MovieDB-API
+Github link: https://github.com/tarunyadav1/Movie-Library-App-using-Reactjs-and-The-MovieDB-API
 
 ### blogs
 
 #### **FIRST BLOG**
 
-    The Birth of React and Reason to learn
-    Link : https://medium.com/@tarunyadav9761/the-birth-of-react-and-reason-to-learn-1a4709e5352c
+The Birth of React and Reason to learn
```
Comment (khusharth) [inline content/participants/ttjc@tarunyadav1/index.md:50]: Please add some description for your blog too.

Comment (tarunyadav1) [reply content/participants/ttjc@tarunyadav1/index.md:58]: @khusharth  Done

File: content/participants/ttjc@tarunyadav1/index.md [hunk 3]
```diff
@@ -1,15 +1,21 @@
 ---
+
```
Comment (khusharth) [inline content/participants/ttjc@tarunyadav1/index.md:2]: You have added space after each tag here. Please remove all those spaces.

Comment (tarunyadav1) [reply content/participants/ttjc@tarunyadav1/index.md]: @khusharth  I had remove space and add more projects. Kindly merge my request.Sorry for late reply.
Thank you

**Comment by khusharth** [comment]
@gkanishk  Good to merge.

**Pull request merged**